manager/controlapi/node_test.go

```diff
@@ -1089,22 +1087,24 @@ func TestRemoveNodeAttachments(t *testing.T) {
 
 	// Now, call the function with our nodeID. make sure it returns no error
 	err = ts.Store.Update(func(tx store.Tx) error {
-		return removeNodeAttachments(tx, "id2")
+		return orphanNodeTasks(tx, "id2")
```

Comment:
the orphaned tasks are deleted by the task reaper, correct? Can we also test for task deletions?